.dockerignore

```diff
@@ -0,0 +1,34 @@
+.git
+.gitignore
+
+# Ignore CI configuration
+.circleci
+
+# Ignore generated files
+**/*.pyc
+
+# Legacy tags are not build
+legacy-*-tags.txt
+
+# Scripts are not executed
+scripts
+
+# Ignore markdown files
+README.md
+docker/README.md
+
+# Ignore Windows specific scripts
+*.bat
+*.ps1
+
+# Ignore Fish scripts
+*.fish
+
+# Ignore tcsh scripts
+*.csh
+
+# Ignore linter configurations
+.flake8
+
+# Makefile is not needed
+Makefile
```

Comment:
Is there some say to inherit everything from .gitignore?      if not perhaps just include it with a comment.

Reply:
Perhaps we could symlink it with `.gitignore`? Anyways, I cleaned up the file a bit with commit https://github.com/emscripten-core/emsdk/pull/571/commits/4b64400017b5d1c49a35b3f330529a1fb797829e.